ContentMigrationBatch#S8WD

simgobb opened the pull request:
https://developers.google.com/web/updates/2019/02/rendering-on-the-web
https://developers.google.com/web/updates/2019/06/pwa-install-addressbar
https://developers.google.com/web/updates/2019/06/pwa-install-patterns
https://developers.google.com/web/fundamentals/design-and-ux/animations
https://developers.google.com/web/fundamentals/design-and-ux/animations/animating-between-views

@rachelandrew

rachelandrew commented:
This one is failing with a duplicate URL, I need to check if we actually have duplicated content or it is different.

rachelandrew commented:
@simgobb can you just delete the one called animations overview? And then redirect the page on web fundamentals to https://web.dev/animations/

simgobb commented:
@rachelandrew I have deleted the file and added the redirect in one of the batches I did yesterday. Should be ok now. 

Let me know if there's anything else holding this up and I'll get to it straightaway.
